controllers/nodehealthcheck_controller.go

```diff
@@ -879,3 +879,27 @@ func updateRequeueAfter(result *ctrl.Result, newRequeueAfter *time.Duration) {
 		result.RequeueAfter = *newRequeueAfter
 	}
 }
+
+func getMinHealthy(spec *remediationv1alpha1.NodeHealthCheckSpec, total int) (int, error) {
+	if spec.MinHealthy != nil && spec.MaxUnhealthy != nil {
```

Comment:
IIUC the validation part is already happening in the webhook.
Do we expect a scenario where the webhook validation will not apply ?

Reply:
Yes, the webhook is supposed to sanify the input and protect from this cases. On the other side a feel a bit coward coward introducing a function that is not properly implementing error checking.

Reply:
Nit: I don't mind double checking but maybe export the validation logic instead of duplicating it.

Reply:
done